.github/labeler.yml

```
@@ -18,6 +18,20 @@
 "Code: Test":
   - tests/**/*
 
+"NPC / Factions":
+  - data/json/npcs/**/*
+
+"CBMs":
```

```suggestion
"Bionics":
```
If you want it to use the existing `Bionics` label

---

Thank you!